ContactDetailsApi.Tests/Dockerfile

```diff
@@ -36,4 +26,3 @@ RUN dotnet build -c Release -o out ContactDetailsApi/ContactDetailsApi.csproj
 RUN dotnet build -c debug -o out ContactDetailsApi.Tests/ContactDetailsApi.Tests.csproj
 
 CMD dotnet test
-RUN dotnet sonarscanner end /d:sonar.login="${SONAR_TOKEN}"
```

Comment:
The sonar scan in this docker-compose is redundant - we have a separate job in CI dedicated to running a sonarcloud scan